packages/venia-concept/src/ServiceWorker/Utilities/htmlHandler.js

```diff
@@ -0,0 +1,17 @@
+const cloneRequestWithDiffURL = (request, url) =>
```

Comment:
Do you see potential uses for this outside of what we have in this PR?
If not, since the only place that calls this always passes in `/` for the `url`, you could just hardcode that here and make this easier to read.

Reply:
I wanted to separate out concerns and have a function just deal with what it needs to and let other utils like clone to take care of cloning a request. But coming to your question, I wasn't sure if this will be useful elsewhere. Took the benefit of the doubt and separated it into its own function but leaving it in the same file. It can be moved and exported later if needed by others.

Reply:
This functionality might be changed a bit in my next PR.